styles/main.css

```diff
@@ -1,7 +1,991 @@
+    width: 100%;
+}
+
+#thumb-five {
+    background-image: url("https://cdnp0.stackassets.com/4ea640481a5cc20e90ba32024b8e6c95f61f448a/store/7976005e114af1b91ceb3a6510caa3d7eba5e9a2e35616d7387a82ddad54/sale_17715_primary_image.jpg");
+    width: 100%;
+}
+
+#thumb-six {
+    background-image: url("https://cdnp0.stackassets.com/32cdfbbcfc023fe935e55ddd400f24e73cf57d51/store/3ce57becc77fb90017437f2ffe5ec803c87b17ab2c6c1727fa204e38f14c/sale_17543_primary_image.jpg");
+    width: 100%;
+}
+
+#thumb-seven {
+    background-image: url("https://cdnp3.stackassets.com/67f53a76e9213d72193acbccabb50af8eb11bd7d/store/6d8c7384f822f13bba37e8e85af7344d04987b49e64774e597f333055112/sale_17729_primary_image.jpg");
+    width: 100%;
+}
+
+#thumb-eight {
+    background-image: url("https://cdnp1.stackassets.com/f1fe66304091f4eab30f4b4f3e151ee648a6a6ea/store/292957d3d8fc9f276bea046313708fc80bac1edd87dd5f9b155e2a617749/sale_17605_primary_image.jpg");
+    width: 100%;
+}
+
+
+/**** FOOTER Section ****/
+footer {
+    height: 17.5rem; /*280px / 16px*/
+    width: 100vw;
+}
+
+.footer-row {
+    min-width: 100%;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.top {
+    background-color: #131313;
+    min-height: 60%; /* 60% of of 280px */
+    min-width: 100%;
+}
+
+.bottom {
+    background-color: black;
+    min-height: 40%; /* */
+    min-width:100%;
+}
+
+.footer-wrapper {
+    min-height:80px; /* 50% of top footer row - must edit this */
+    min-width: 100%;
+    color: #888;
+    align-items: center;
+}
+#footer-social .fa-circle {
+    color: #333;
+}
+
+#footer-social .fa-stack:hover .fa-circle {
+    color: #131313;
+}
+
+#footer-nav {
+    margin-top: 2rem;
+}
+
+#footer-nav .footer-nav-link {
+    text-transform: uppercase;
+    color: #888;
+    font-weight: 400;
+    font-size: 0.8rem;
+    margin: 0 0.5rem;
+}
+
+#footer-nav .footer-nav-link:hover {
+    color: #fff;
+}
+
+.footer-wrapper span img {
+    height: 9px;
+}
+
+.footer-wrapper span {
+    font-size: 13px;
+}
+
+#heart {
+    color: red;
+}
+
+/******  Width >= 786px < 1024 ******/
+
+@media only screen and (min-width: 768px) {
+    /* Navigation */
+    header {
+        grid-template-rows: 1fr 50px 1fr;
+        height: 6.875rem;
+    }
+
+    #brand {
+        grid-row: span 2;
+    }
+
+    #menu {
+        display: none;
+    }
+
+    nav {
+        grid-template-rows: 1fr 50px;
+        grid-row: span 2;
+    }
+
+    #search {
+        grid-column: -1;
+    }
+
+    #topnav {
+        display: flex;
+    }
+
+    #basenav {
+        color: #aaa;
+        display: flex;
+        grid-row: 2;
+    }
+
+    #basenav li:nth-child(n+6) {
+        display: none;
+    }
+
+    .seme {
+        grid-row: 2;
+        align-self: center;
+    }
+
+    /* End Navigation */
+
+    /* cover*/
+    #cover {
+        height: 60vh;
+        flex-direction: row;
+        flex-flow: column wrap;
```

Comment:
This can conflict with the previous line

Reply:
Could you please expand on this comment, we'd like to understand what you mean by it?